### ISSUE TITLE
ARM64 builds support

### ISSUE THREAD

**Issue by ShutdownRepo** [ThePorgs/Exegol-images#9]
**Configuration**: MacBook Pro M1 Pro, macOS 12.3.1
**Exegol install**: wrapper v4.0.2b1, nightly image v.3cc234ea

On a standard nmap scan, it crashes with the following message.

```
# nmap -v -T4 -p 389 10.10.11.129
Starting Nmap 7.92 ( https://nmap.org ) at 2022-05-19 19:19 CEST
Initiating Ping Scan at 19:19
Scanning 10.10.11.129 [4 ports]
pcap_create(tun0) FAILED: tun0: SIOCETHTOOL(ETHTOOL_GET_TS_INFO) ioctl failed: Function not implemented.
Call to pcap_open_live() failed three times. There are several possible reasons for this, depending on your operating system:
LINUX: If you are getting Socket type not supported, try modprobe af_packet or recompile your kernel with PACKET enabled.
 *BSD:  If you are getting device not configured, you need to recompile your kernel with Berkeley Packet Filter support.  If you are getting No such file or directory, try creating the device (eg cd /dev; MAKEDEV <device>; or use mknod).
*WINDOWS:  Nmap only supports ethernet interfaces on Windows for most operations because Microsoft disabled raw sockets as of Windows XP SP2.  Depending on the reason for this error, it is possible that the --unprivileged command-line argument will help.
SOLARIS:  If you are trying to scan localhost or the address of an interface and are getting '/dev/lo0: No such file or directory' or 'lo0: No DLPI device found', complain to Sun.  I don't think Solaris can support advanced localhost scans.  You can probably use "-Pn -sT localhost" though.


QUITTING!
```

**Comment by ShutdownRepo** [comment]
This is due to the image architecture.

Exegol needs multi-arch images.

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/40902872/169373512-9fe5d4b2-74a6-4e95-95f3-a43dc8ba189f.png">
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/40902872/169373578-42390913-c730-4b57-97e8-c29aa04a5467.png">

**Comment by ShutdownRepo** [comment]
The issue should be fixed from 1efeb9f9e1e50ee7c6d9ac082223d2fae03cd4be
Exegol images can now be built for AMD64 or ARM64 platforms (Apple Silicon M1 processors are ARM64 CPUs).
While the various CI/CD configured for Exegol don't build multi-processor images yet, an ARM64 image can be built locally with the following command: `exegol install local`. The `arm64` profile can then be chosen.
The arm64 profile is the same thing as the `full`one, but the `linux/arm64`platform is specified.

**Comment by ShutdownRepo** [comment]
Fixed in #77
